container/features/src/main/resources/features.xml

```diff
@@ -1,1333 +1,1155 @@
-    </feature>
-
-    <feature name="commons-beanutils" description="Apache :: commons-beanutils" version="${commonsBeanutilsVersion}">
-      <feature>commons-collections</feature>
-
-      <bundle>mvn:commons-beanutils/commons-beanutils/${commonsBeanutilsVersion}</bundle>
-    </feature>
-
-    <feature name="commons-digester" description="Apache :: commons-digester" version="${commonsDigesterVersion}">
-      <feature>commons-beanutils</feature>
-
-      <bundle>mvn:commons-digester/commons-digester/${commonsDigesterVersion}</bundle>
-    </feature>
-
-    <feature name="commons-exec" description="Apache :: commons-exec" version="1.2">
-      <bundle>mvn:org.apache.commons/commons-exec/1.2</bundle>
-    </feature>
-
-    <feature name="commons-io" description="Apache :: commons-io" version="${commonsIoVersion}">
-      <bundle>mvn:commons-io/commons-io/${commonsIoVersion}</bundle>
-    </feature>
-
-    <feature name="commons-jexl" description="Apache :: commons-jexl" version="${commonsJexlVersion}">
-      <bundle>mvn:org.apache.commons/commons-jexl/${commonsJexlVersion}</bundle>
-    </feature>
-
-    <feature name="commons-jxpath" description="Apache :: commons-jxpath" version="${commonsJxpathVersion}">
-      <feature>commons-beanutils</feature>
-      <feature>javax.servlet</feature>
-
-      <!-- jdom has OSGi issues... use the ServiceMix bundle for it -->
-      <!-- @see https://developer.atlassian.com/docs/faq/plugin-framework-faq/using-jdom-in-osgi -->
-      <!-- @see https://techotom.wordpress.com/2014/10/21/fixing-the-default-package-is-not-permitted-by-the-import-package-syntax-with-maven-bundle-plugin/ -->
-      <!-- <bundle>wrap:mvn:jdom/jdom/1.0</bundle> -->
-      <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.jdom/1.1_4</bundle>
-      <bundle dependency="true">wrap:mvn:javax.servlet.jsp/jsp-api/2.2</bundle>
-
-      <bundle>mvn:commons-jxpath/commons-jxpath/${commonsJxpathVersion}</bundle>
-    </feature>
-
-    <feature name="commons-lang" description="Apache :: commons-lang" version="${commonsLangVersion}">
-      <bundle>mvn:commons-lang/commons-lang/${commonsLangVersion}</bundle>
-    </feature>
-
-    <feature name="commons-lang3" description="Apache :: commons-lang3" version="${commonsLang3Version}">
-      <bundle>mvn:org.apache.commons/commons-lang3/${commonsLang3Version}</bundle>
-    </feature>
-
-    <feature name="commons-net" description="Apache :: commons-net" version="3.6">
-      <bundle>mvn:commons-net/commons-net/3.6</bundle>
-    </feature>
-
-    <feature name="dnsjava" description="dnsjava" version="${dnsjavaVersion}">
-      <bundle dependency="true">wrap:mvn:org.dnsjava/dnsjava/${dnsjavaVersion}</bundle>
+    <repository>mvn:org.apache.activemq/activemq-karaf/${activemqVersion}/xml/features</repository>
+    <!--TODO: This could be moved to the full assembly pom-->
```

Comment:
TODO

Reply:
I'm in favor of removing this mega feature and declaring the features to be pulled into the system directory in OpenNMS directly in the full assembly pom instead.